ngraph/core/src/pass/low_latency.cpp

```diff
@@ -6,12 +6,26 @@
 
 #include <memory>
 
+#include <ngraph/log.hpp>
 #include <ngraph/opsets/opset6.hpp>
+#include <ngraph/opsets/opset7.hpp>
 #include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
 #include <ngraph/variant.hpp>
 
+NGRAPH_RTTI_DEFINITION(ngraph::pass::LowLatency2, "LowLatency2", 0);
+
+NGRAPH_SUPPRESS_DEPRECATED_START
 NGRAPH_RTTI_DEFINITION(ngraph::pass::LowLatency, "LowLatency", 0);
 
+using namespace std;
+using namespace ngraph;
+
+string generate_variable_name(const string& op_name, const string& param_name, int variable_idx)
```

Comment:
please add this method to anonymous namespace.

Reply:
done